package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.5.9",
+  "version": "3.5.10",
```

Comment:
This PR isn't only unit tests so I updated this app version -- so we can tell the difference if there's a bug in this version but not in the previous one.